frontend/src/components/History/HistoryEditor/HistoryEditor.tsx

```diff
@@ -30,7 +30,8 @@ export default function HistoryEditor({
 }: IOwnProps): JSX.Element {
   const [updatedAttempt, setUpdatedAttempt] = useState<Attempt>(attempt);
   const toast = useToast();
-  const token = useJwt();
+  const auth = useAuth();
+  const token = auth.token ?? "";
```

Comment:
seems like every component that uses `auth.token` has this line of code to set to default of empty string if token is null. would it be a good idea to set the default value of `token` to `""` so that the components who consume `auth.token` doesn't have to check if its null and set the default to `""`?

Reply:
Yeah, it's easier to just do that for now instead of correcting all the fetch functions to accept `null`.